package.json

```diff
@@ -30,7 +31,9 @@
   "devDependencies": {
     "audio-generator": "^1.0.5",
     "audio-speaker": "^1.1.2",
-    "watchify": "^3.6.0"
+    "mic": "^2.1.1",
+    "watchify": "^3.6.0",
```

Comment:
@mreinstein why budo is complex? I mean we should not add it to dependency of course, but `budo test` for me is pretty simple :smile: